packages/yswaps/scripts/Router.ts

```diff
@@ -0,0 +1,57 @@
+import { PendingTrade, TradeSetup } from './types';
+import uniswap from '@libraries/uniswap-v2';
+import { UNISWAP_V2_FACTORY, UNISWAP_V2_ROUTER, WETH } from '@deploy/mainnet-swappers/uniswap_v2';
+import { ethers, getChainId } from 'hardhat';
+import zrx from './libraries/zrx';
+
+export class Router {
```

Comment:
This should be a library, not a class

Reply:
a library is an file with functions, right?